### ISSUE TITLE
add copy ip button to ips in `Ip Addresses` column on interface lists

### ISSUE THREAD

**Issue by ITJamie** [netbox-community/netbox#13697]
### NetBox version

v3.6.0

### Feature type

Change to existing functionality

### Proposed functionality

when looking at a list of interfaces on a device / VM it would be nice to have the quick copy IP button that has appeared elsewhere recently 

### Use case

ability to copy ip from device/vm interface list without having to select first goto the IP object page

### Database changes

n/a

### External dependencies

n/a

**Comment by jeremystretch** [comment]
Marking this as blocked by #12128 for now as we've made changes to the affected column template code.

**Comment by goteamkor** [comment]
If https://github.com/netbox-community/netbox/issues/17229 were implemented I think it would accomplish what this FR is asking for.
